src/runtime_src/core/pcie/tools/xbmgmt/xospiversal.cpp

```diff
@@ -43,7 +43,8 @@ int XOSPIVER_Flasher::xclUpgradeFirmware(std::istream& binStream)
     binStream.seekg(0, binStream.beg);
 
     std::cout << "INFO: ***PDI has " << total_size << " bytes" << std::endl;
-    int fd = mDev->open("xfer_versal", O_RDWR);
+    //int fd = mDev->open("xfer_versal", O_RDWR);
+    int fd = mDev->open("xgq", O_RDWR);
```

Comment:
Change this

Reply:
Yes, I will. 
Could the xbmgmt2 issue be fixed? Otherwise, they cannot use mast XRT to flash the VMR based shell.

Reply:
Yes, I'll work on it today

Reply:
fixed